### PR TITLE
Redesigned WebRTC welcome screen

### PR DESCRIPTION
**Pull request by idzkowski-google**

### CODE REVIEW

File: WebRTCSample/app/res/layout/glass_activity_connect.xml [hunk 1]
```diff
@@ -1,36 +1,63 @@
...
-        android:layout_width="match_parent"
-        android:layout_height="match_parent"
-        android:gravity="center"
-        android:orientation="vertical">
+    <TextView
+        android:id="@+id/textView2"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:fontFamily="sans-serif-light"
+        android:text="@string/navigate_to"
```
Comment (jbolinger) [inline WebRTCSample/app/res/layout/glass_activity_connect.xml:13]: Did you forget to app these? Looks like that's why the build failed.

Comment (idzkowski-google) [reply WebRTCSample/app/res/layout/glass_activity_connect.xml:13]: I'm sorry I missed that. Fixed in the next commit.

Comment (jbolinger) [reply WebRTCSample/app/res/layout/glass_activity_connect.xml:13]: no worries, that's why we have the checks.

File: WebRTCSample/app/res/values/strings.xml [hunk 2]
```diff
@@ -223,8 +223,11 @@
     <string name="pref_use_legacy_audio_device_title">Use legacy audio device.</string>
     <string name="pref_use_legacy_audio_device_default">false</string>
     <string name="room_name_label">Room name</string>
-    <string name="connect_on_another_device">Connect to https://appr.tc/ \n on another device \n and enter room name:</string>
+    <string name="connect_on_another_device">Connect to https://appr.tc/ on another device and enter room name:</string>
```
Comment (jbolinger) [inline WebRTCSample/app/res/values/strings.xml:226]: looks like you can delete this now?

Comment (idzkowski-google) [reply WebRTCSample/app/res/values/strings.xml]: Removed in the next commit.

**Pull request merged**